src/compute-client/src/controller/instance.rs

```diff
@@ -623,6 +623,34 @@ impl<T: ComputeControllerTimestamp> Instance<T> {
         );
     }
 
+    /// Returns `true` if the given collection is hydrated on at least one
+    /// replica.
+    ///
+    /// This also returns `true` in case this cluster does not have any
+    /// replicas.
+    #[mz_ore::instrument(level = "debug")]
+    pub fn collection_hydrated(
+        &self,
+        collection_id: GlobalId,
+    ) -> Result<bool, HydrationCheckBadTarget> {
```

Comment:
It looks like this function never returns `Err` or maybe I'm just missing something.

Reply:
thanks! fixed that one by changing the `expect()` to return the error